packages/grid/src/BaseCell.css

```diff
@@ -27,3 +27,86 @@
 .saltGridCell-editable:focus .saltGridBaseCell-valueContainer {
   border-color: transparent;
 }
+
+.saltGridCell-editable .saltGridBaseCell-valueContainer-status-error,
+.saltGridBaseCell-status-error:focus .saltGridCursor {
+  border-color: var(--salt-status-error-borderColor);
+}
+
+.saltGridBaseCell-status-error:focus .saltGridCornerTag-focusOnly {
+  border-left-color: var(--salt-status-error-borderColor);
+}
+
+.saltGridBaseCell-valueContainer-status-error {
+  background-color: var(--salt-palette-error-background-emphasize);
+}
+
+.saltGridCell-editable .saltGridBaseCell-valueContainer-status-success,
+.saltGridBaseCell-status-success:focus .saltGridCursor {
+  border-color: var(--salt-status-success-borderColor);
+}
+
+.saltGridBaseCell-status-success:focus .saltGridCornerTag-focusOnly {
+  border-left-color: var(--salt-status-success-borderColor);
+}
+
+.saltGridBaseCell-valueContainer-status-success {
+  background-color: var(--salt-palette-success-background-emphasize);
+}
+
+.saltGridCell-editable .saltGridBaseCell-valueContainer-status-warning,
+.saltGridBaseCell-status-warning:focus .saltGridCursor {
+  border-color: var(--salt-status-warning-borderColor);
+}
+
+.saltGridBaseCell-status-warning:focus .saltGridCornerTag-focusOnly {
+  border-left-color: var(--salt-status-warning-borderColor);
+}
+
+.saltGridBaseCell-valueContainer-status-warning {
+  background-color: var(--salt-palette-warning-background-emphasize);
+}
+
+.salt-density-high {
+  --grid-statusIndicator-size: 6px;
+}
+
+.salt-density-medium {
+  --grid-statusIndicator-size: 8px;
+}
+
+.salt-density-low {
+  --grid-statusIndicator-size: 10px;
+}
+
+.salt-density-touch {
+  --grid-statusIndicator-size: 12px;
+}
+
+.saltGridBaseCell-statusContainer {
+  font-size: var(--saltGrid-statusIndicator-size, var(--grid-statusIndicator-size));
+  position: absolute;
+  bottom: 4px;
+  right: 4px;
```

Comment:
will these be affected by density? could use --salt-size-basis-unit here if not

Reply:
Have to double check in figma

Reply:
It needs to be hardcoded